fractions/microprofile/microprofile-restclient/README.adoc

```diff
@@ -1,4 +1,4 @@
-= MicroProfile Rest Client
+= Eclipse MicroProfile Rest Client
```

Comment:
Same here. We can add 1 sentence below that would use the full name, similarly to how other fractions have it.